ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -606,7 +624,8 @@ open class DrawerController: UIViewController {
     private func updatePreferredContentSize(isExpanded: Bool) {
         isPreferredContentSizeBeingChangedInternally = true
         if isExpanded {
-            preferredContentSize.height = UIScreen.main.bounds.height
+            let expandedContentHeight = (preferredMaximumExpansionHeight < UIScreen.main.bounds.height && preferredMaximumExpansionHeight >= originalDrawerHeight) ? preferredMaximumExpansionHeight : UIScreen.main.bounds.height
```

Comment:
nit: (1)we really should shop using uiscreen.main.bounds. we should just use the current windows bound instead. (2) this value is used multiple times we should catch it?

Reply:
i am not how to find the current windows bounds. Can you help with this? I will update it.